User Guide - Add Usergroup attributes (#8907)
geosolutions-it/MapStore2: docs/user-guide/managing-groups.md

Through the *Members manager* section it is possible to choose which users are part of the group:

<img src="../img/managing-groups/sel-users.gif" class="ms-docimage"/>

## Attributes

> On the *Attributes* tab the admin can associate attributes to user group. By default MapStore allows to enter a "notes" attribute for each group. 

Reviewer: ```suggestion
On the *Attributes* tab the admin can associate attributes to user group. By default MapStore allows to enter a "notes" attribute for each group.
```

> 

Reviewer: ```suggestion
```

Through the *Members manager* section it is possible to choose which users are part of the group:

<img src="../img/managing-groups/sel-users.gif" class="ms-docimage"/>

## Attributes

> On the *Attributes* tab the admin can associate attributes to user group. By default MapStore allows to enter a "notes" attribute for each group.

Reviewer: ```suggestion
On the *Attributes* tab the admin can associate some attributes to user groups. By default MapStore allows to enter a "notes" attribute for each group. The attributes list can be configured by editing the [plugin configuration](https://mapstore.geosolutionsgroup.com/mapstore/docs/api/plugins#plugins.GroupManager) in `localConfig.json`. 
```

> On the *Attributes* tab the admin can associate attributes to user group. By default [Mapstore](https://mapstore.geosolutionsgroup.com/mapstore/#/) allows to enter a notes as attribute for a group.

Reviewer: No need to link mapstore demo here. 
```suggestion
On the *Attributes* tab the admin can associate attributes to user group. By default MapStore allows to enter a "notes" attribute for each group. 

```
